test/fixtures/get-orders-response.json

```diff
@@ -7,9 +7,9 @@
         "value": "17.70155237781915",
         "counterparty": "rMwjYedjc7qqtKYVLiAccJSmCwih4LnE2q"
       },
-      "totalPrice": {
+      "price": {
```

Comment:
Yeah. I'm not sure which is better, having consistency with orderbook orders by showing maker or reducing the redundancy by not showing the same maker in all the results. Do you feel strongly either way?